packages/sensors/lib/sensors.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+const EventChannel _accelerometerEventChannel =
+    const EventChannel('plugins.flutter.io/accelerometer');
+
+const EventChannel _gyroscopeEventChannel =
+    const EventChannel('plugins.flutter.io/gyroscope');
+
+class AccelerometerEvent {
```

Comment:
Instances should be unmodifiable, with `final` fields. Same for `GyroscopeEvent`.